src/main/kotlin/no/nav/syfo/senoppfolging/v2/SenOppfolgingControllerV2.kt

```diff
@@ -96,8 +99,8 @@ class SenOppfolgingControllerV2(
             responseStatus = response?.questionResponses?.toResponseStatus() ?: ResponseStatus.NO_RESPONSE,
             response = response?.questionResponses,
             responseTime = response?.createdAt?.format(DateTimeFormatter.ofPattern("dd.MM.yyyy")),
-            maxDate = sykepengerMaxDateResponse?.maxDate,
-            gjenstaendeSykedager = sykepengerMaxDateResponse?.gjenstaendeSykedager,
+            maxDate = sykepengerInformasjon?.forelopigBeregnetSlutt?.let { formatDateForLetter(it) },
```

Comment:
En ting som er litt nice i kotlin er extension functions, som gjør at man kan slippe en del duplikat kode og få litt mer lesbarhet. Her kunne du ha vurdert å lagt til en slik funksjon i PSykepengedagerInformasjon, ala 
`fun PSykepengedagerInformasjon.forelopigBeregnetSluttFormatted(): String {
    return formatDateForLetter(this.forelopigBeregnetSlutt)
}`

Så kan man bruke det slik: `sykepengerInformasjon?.forelopigBeregnetSluttFormatted()` alle steder det trengs.

Foretrekker ofte det fremfor util metoder :)

Reply:
Kult! Prøver det.

Reply:
Nå har jeg gjort det, hvis du vil ta en kikk.